data/io.github.finefindus.Hieroglyphic.metainfo.xml.in.in

```diff
@@ -18,6 +18,12 @@
     <p>If you work with LaTeX, you know it's difficult to memorize the names of all the symbols. Hieroglyphic allows you to search through over 1000 different LaTeX symbols by sketching.</p>
   </description>
 
+  <categories>
+    <category>Math</category>
+    <category>Science</category>
+    <category>Utility</category>
+  </categories>
+
```

Comment:
What is the advantage of defining them here again? Since [they are already extracted from the desktop file](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#categories-and-keywords)?